Source/ACE.Server/WorldObjects/Player_Inventory.cs

```diff
@@ -2327,11 +2327,18 @@ private void HandleIOUTurnIn(WorldObject target, WorldObject iouToTurnIn)
             Session.Network.EnqueueSend(new GameMessageSystemChat($"You allow {target.Name} to examine your {iouToTurnIn.NameWithMaterial}.", ChatMessageType.Broadcast));
             Session.Network.EnqueueSend(new GameEventInventoryServerSaveFailed(Session, iouToTurnIn.Guid.Full, WeenieError.TradeAiRefuseEmote));
 
-            if (iouToTurnIn is Book book)
+            if (!PropertyManager.GetBool("iou_trades").Item)
+            {
+                //Session.Network.EnqueueSend(new GameMessageHearDirectSpeech(target, "Hmm... Something isn't quite right with this IOU. I can't seem to make out what its for. I'm sorry!", this, ChatMessageType.Tell));
+                Session.Network.EnqueueSend(new GameEventWeenieErrorWithString(Session, (WeenieErrorWithString)WeenieError.TradeAiDoesntWant, target.Name));
+                return;
+            }
+
+            if (iouToTurnIn is Book book && book.ScribeName == "ACEmulator" && book.ScribeAccount == "prewritten")
```

Comment:
what happens when someone creates account 'prewritten', and character name 'ACEmulator'?

Reply:
it continues on this line
`if (page != null && page.AuthorName == "ACEmulator" && page.AuthorAccount == "prewritten" && page.AuthorId == uint.MaxValue)` and fails